human-forecasts/performance-board/app.R

```diff
@@ -0,0 +1,180 @@
+library(shiny)
+library(magrittr)
+library(ggplot2)
+library(scoringutils)
+
```

Comment:
Deps missing including dplyr and readr. Likely has no impact on reuse as listed elsewhere but generally good practice to up include the full list or nothing at the top of the script.